plugins/auth-backend/src/service/OidcService.ts

```diff
@@ -403,7 +404,7 @@ export class OidcService {
     return {
       accessToken: token,
       tokenType: 'Bearer',
-      expiresIn: 3600,
+      expiresIn: expiresIn ?? 3600,
```

Comment:
So there's a slight issue here.

As you might have noticed above, the original `readTokenExpiration` call is used to set how long the tokens that are issued from the `TokenIssuer` are actually valid for as they have also `exp` claims inside the token.

The issue with only changing this value and increasing from the default is that the token would not be valid unless you updated both.

I guess we have a few options here, we either create a new `TokenIssuer` for this which feels a little bit unfortunate, we refactor the `TokenIssuer` interface to take an `expiry` when calling `issueToken` or we just use the original value from config in this expiry block, and don't treat them as two different config values at all?

Wdyt?

Also gonna bring in @Rugvip for some other opinions here too.

Reply:
If we want it to be separate, then I guess we'd need to wire up a new option for `issueToken` like you said